charts/fission-all/templates/router/secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.authentication.enabled }}
+apiVersion: v1
```

Comment:
We can use `pre-install` hook for this resource ?
https://helm.sh/docs/topics/charts_hooks/